src/main/scala/introcourse/level01/IntroExercises.scala

```diff
@@ -86,6 +86,11 @@ object IntroExercises {
 
   /**
     * How can we extract the first element of a pair?
+    *
+    * Hint 1: Check the getters on `pair`.
+    * Hint 2: Read up on pattern matching on tuples.
+    *
+    * https://docs.scala-lang.org/tour/tuples.html
     */
   def fst(pair: (String, Int)): String = ???
```

Comment:
is there any reason we don't call this `first`?  Is it worth saving 2 characters?

Reply:
I'll rename `fst` and `snd`.

The Haskell equivalents are `fst` and `snd` but that's not a good reason.